pygments/lexers/tal.py

```diff
@@ -0,0 +1,58 @@
+"""
+    pygments.lexers.uxntal
+    ~~~~~~~~~~~~~~~~~~~~~~
```

Comment:
```suggestion
    pygments.lexers.tal
    ~~~~~~~~~~~~~~~~~~~
```

This should in theory make `make check` happy, but it also apparently can't find `pyflakes`, which seems odd …